addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -185,6 +185,9 @@ class ComposerTextInput extends Component {
      */
     _updateHeight() {
         this._mirroredTextareaRef.el.value = this.composer.textInputContent;
+        // In firefox, if you fill the first line of the composer completely and hit "enter"
+        // the scrollbar will appear (before we reach max-height)
+        this._textareaRef.el.style.height = "0px";
```

Comment:
Hello, thank you for the PR;

This appear to fix the scroll issue correctly, but this would revert the change that was done in 52f3db8ef61d4284d8ea624022bc1355f7f8cf00 and it willreintroduce the other issue.
If you test in community with a chatter that is below the form, and you press enter many times, you will see the whole page scroll is reset.

Do you see any other way to resolve it?

Reply:
@seb-odoo . Actually with my changes actually I unable to reproduce the issue. Can you please elaborate more on this?

And I can not find any other way specific to firefox, as any of the changes regarding firefox will affect other browsers too.